src/components/inputs/Slider/SliderInputArrow.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export const SliderInputArrow = (props) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="12"
+    height="10"
+    viewBox="0 0 12 10"
+    fill="none"
+    {...props}
+  >
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M6.00178 4.91424L3.70689 7.2074L2.29321 5.79265L5.29547 2.79265C5.48308 2.60519 5.73747 2.49993 6.00269 2.50002C6.2679 2.50012 6.52222 2.60558 6.70968 2.79318L9.70743 5.79318L8.29268 7.20687L6.00178 4.91424Z"
+      fill="#11191D"
```

Comment:
I don't see the arrows in dark mode, that could probably be fixed with assigning this as `currentColor` and modifying the `color` in the `<ArrowsContainer>`.